lib/cc/services/jira.rb

```diff
@@ -3,7 +3,7 @@
 class CC::Service::Jira < CC::Service
   class Config < CC::Service::Config
     attribute :domain, Axiom::Types::String,
-      description: "Your JIRA host domain (e.g. yourjira.com:PORT, please exclude https://)"
+      description: "Your JIRA host domain (e.g. yourjira.com[:PORT], please exclude https://. Include PORT only if you are using a port number to access your own server)"
```

Comment:
```suggestion
      description: "Your JIRA host domain (e.g. yourjira.com[:PORT], please exclude "https://" and only  add PORT if you are using a different port number to access your own server that is not the default)"
```

Reply:
I think that even if you are using the default port, you need to include the port number here. You don't need the port number if you are not hosting your own Jira instance or if you are but have a DNS configured so you can access the instance just by its domain.

Reply:
PORT follows its protocol, for https it is 443 - it is the only use I see, so if not specified, 443 is what it is gonna be, right?

Reply:
feel free to drop it if  it made things more confusing.  🤦